app/src/main/java/com/hedvig/app/util/featureflags/loginmethod/LoginMethodProvider.kt

```diff
@@ -3,5 +3,5 @@ package com.hedvig.app.util.featureflags.loginmethod
 import com.hedvig.hanalytics.LoginMethod
 
 interface LoginMethodProvider {
-  suspend fun getLoginMethod(): LoginMethod
+  suspend fun getLoginMethod(): LoginMethod?
```

Comment:
Why was thing change necessary? Just so I understand what has changed (or what was wrong before)

Reply:
Its because `DevLoginMethodProvider` checks `marketManager`, and we have cases were the market could be null (eg on first install). This was not the case before though, perhaps we initialised the market somewhere else. I defaulted to SE login method for the dev provider now instead of throwing, so we can still return non nullable login method. Good catch again :)